.github/workflows/main.yml

```diff
@@ -27,17 +27,18 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Install NEP29 dependencies
+      if: ${{ matrix.numpy_ver != 'latest'}}
+      run: |
+        pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
+        pip install "xarray<2022.06"
```

Comment:
xarray limit only required for NEP29 case.

Reply:
do we still need this restriction?

Reply:
We need this for the NEP29 case since the old numpy is not compatible with the latest xarray.